combine.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+shopt -s globstar
+
+path="/home/$(logname)/pikaraoke-songs"
```

Comment:
Assumes the user is downloading to pikaraoke-songs, which may not be the case if they change the download dir.

Reply:
Maybe I should have it take in whatever the user sets for the -d option, and ~/pikaraoke-songs otherwise.

In that case, I'd insert a call to the script in app.py. will revise.